pallets/collator-assignment/src/lib.rs

```diff
@@ -177,10 +182,43 @@ pub mod pallet {
                 let mut rng: ChaCha20Rng = SeedableRng::from_seed(random_seed);
                 collators.shuffle(&mut rng);
                 container_chain_ids.shuffle(&mut rng);
+                parathreads.shuffle(&mut rng);
             }
 
             // We read current assigned collators
             let old_assigned = Self::read_assigned_collators();
+            // Initialize list of container chains as `[orchestrator, container1, container2, parathread1, parathread2]`.
+            // The order means priority: the first chain in the list will be the first one to get assigned collators.
+            // Chains will not be assigned less than `min_collators`, except the orchestrator chain.
+            // First all chains will be assigned `min_collators`, and then the first one will be assigned up to `max`,
+            // then the second one, and so on.
+            let mut container_chains = vec![ContainerChain {
```

Comment:
I am not entirely sure I like calling this a containerChain when it really isnt :(. I would rather call it Chain, but I also dont like it too much, maybe we can have a dual enum here that for now have the same parameters?

Reply:
Renamed to `ChainNumCollators` and added docs saying that this could be any kind of chain